.gitlab-ci.yml

```diff
@@ -1391,6 +1391,15 @@ release_docker_images:
         docker push $docker_url:${version};
       done
 
+release_docker_images:client_only:
```

Comment:
There is already a `publish_docker_client_images` job. Why don't you reuse that?

You could perhaps bake it into #435, since it would follow a similar style with one button and one variable based trigger (the already existing `PUBLISH_DOCKER_CLIENT_IMAGES`).

Reply:
I see, I wasn't sure if that was the job I wanted. I'm just gonna close this I think.

Reply:
Using this PR for the entire client-only release